internal/provider/schema_util.go

```diff
@@ -66,6 +66,41 @@ func MustAddMountMigrationSchema(r *schema.Resource, customStateUpgrade bool) *s
 	return r
 }
 
+func MustAddSecretsSyncCommonSchema(r *schema.Resource) *schema.Resource {
+	MustAddSchema(r, map[string]*schema.Schema{
+		consts.FieldType: {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Type of secrets destination.",
+			ForceNew:    true,
+		},
+		consts.FieldSecretNameTemplate: {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+			Description: "Template describing how to generate external secret names.",
+			// @TODO can this be updated?
```

Comment:
@maxcoulombe are `secret_name_template` and `custom_tags` editable? I left them as non-editable for now, following the document you had sent over regarding editable fields and environment variables, but since these fields were added in later I figured it could be that the document had not been updated yet

Reply:
They should be editable.

Reply:
+1 they are editable